simulation/tests/test_agents.py

```diff
@@ -473,3 +473,22 @@ def test_heat_pumps_never_selected_if_hassle_factor_is_one_and_not_currently_ins
             household.choose_heating_system(costs, heating_system_hassle_factor=1)
             == HeatingSystem.BOILER_GAS
         )
+
+    @pytest.mark.parametrize("heating_system", list(HeatingSystem))
+    def test_households_install_working_heating_systems_at_model_current_datetime(
+        self,
+        heating_system,
+    ) -> None:
+
+        household = household_factory(
+            heating_system=random.choices(list(HeatingSystem))
```

Comment:
Would `random.choices(list(...))` not return a list? Do we not need a `[0]` at the end to ensure it takes a singleton item? I think we kind of get away with it here since we arent checking the initial heating system value at any point in the method or the test.